### ISSUE TITLE
ToolTipDefinition needs to be dismissible, hoverable and persistent (WCAG 2.1 Checkpoint 1.4.13)

### ISSUE THREAD

**Issue by ajlohr** [carbon-design-system/carbon-components-react#2076]
## Environment

> Operating system

macOS 10.13.3 (17D47)

> Browser

Chrome Version 73.0.3683.75 (Official Build) (64-bit)

> Automated testing tool and ruleset

IBM Accessibility February 2019

> Assistive technology used to verify

Manual check

## Detailed description

[WCAG 2.1 Checkpoint 1.4.13 Content on Hover Focus](https://www.ibm.com/able/guidelines/ci162/content_on_hover_71.html) says when additional content is popped up on hover it must be dismissible, hoverable and persistent. 

1. dismissible
   * When I hover over the popup there is no way to dismiss it using the ESC key. To comply there has to be another way then moving the mouse or tabbing the keyboard focus out
1.  hoverable
   * When I try to hover over the popup it disappears instead of staying visible
1. Persistent
   * I think its already okay with this one, but would be nice to double check

> What version of the Carbon Design System are you using?

```
        "carbon-components": "9.84.14",
        "carbon-components-react": "6.108.0",
        "carbon-icons": "7.0.7",
```

> What did you expect to happen?

tooltip to be dismissed by clicking ESC key or similar and be persistent when moving mouse over content

> What happened instead?

tooltip popup stayed when ESC was pressed, and popup disappears when trying to hover over the text body

> What WCAG 2.1 checkpoint does the issue violate?

WCAG 2.1 Checkpoint 1.4.13- Content on Hover Focus.

## Steps to reproduce the issue

1. Add ToolTipDefinition and hover over tooltip, try to dismiss the popup without moving mouse or using keyboard focus (tab).

Then try to move the mouse over the popup text and the popup disappears when it should be persistent.

**Comment by Jaorji** [comment]
I'm in the pipeline team, we fond that when we click the hover tooltip, it will disappear.

**Comment by dakahn** [comment]
Hey -- doing some testing. Looks to me like the concerns here are addressed in the latest. 🎉 Happy to open up again if there's something I'm missing.
